scripts/intl/intl-zanata-pull.sh

```diff
@@ -1,9 +1,10 @@
 #! /usr/bin/env bash
 [[ -e package.json ]] || cd ../..
 
-JAVA_HOME=$(alternatives --list | grep 'jre_1.8.0[[:space:]]' | cut -f 3)
+JAVA_HOME=$(alternatives --list | grep 'jre_1.8.0' | head -1 | cut -f 3)
+ZANATA_LOCALES=de,es,fr,it,ja,ko,pt-BR,zh-CN,cs
 
-zanata-cli pull \
-    --errors \
-    --pull-type trans \
-    --locales de,es,fr,it,ja,ko,pt-BR,zh-CN,cs
+mvn \
```

Comment:
After running on my machine the script I've received:
```
[INFO] Loading project config from /home/user/ovirt-web-ui/zanata.xml
User config file '/home/user/.config/zanata.ini' not found; ignoring.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 33.797 s
[INFO] Finished at: 2020-10-14T10:38:39+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.zanata:zanata-maven-plugin:4.6.2:pull (default-cli) on project standalone-pom: Zanata mojo exception: RESTEASY004655: Unable to invoke request: NullPointerException -> [Help 1]
[ERROR] 
```

I have Java 8 and it seems detected correctly so I guess this is due to missing `zanata.ini`, right?

Reply:
> I have Java 8 and it seems detected correctly so I guess this is due to missing zanata.ini, right?

Yes.  I renamed my zanata.ini and had the same fail as you:

```
 sdickers@ghost  ~/projects/sjd78-ovirt-web-ui  mv ~/.config/zanata.ini{,_}
 sdickers@ghost  ~/projects/sjd78-ovirt-web-ui  yarn intl:extract
yarn run v1.22.5
$ scripts/intl/intl-extract.sh
> [extract-messages.js] write file -> extra/to-zanata/messages.json ✔️

> [extract-messages.js] write file -> extra/to-zanata/time-durations.json ✔️

> [react-intl-po] write file -> extra/to-zanata/messages.pot ✔️

> [react-intl-po] write file -> extra/to-zanata/time-durations.pot ✔️

Done in 1.18s.

 sdickers@ghost  ~/projects/sjd78-ovirt-web-ui  yarn intl:pull
yarn run v1.22.5
$ scripts/intl/intl-zanata-pull.sh
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- zanata-maven-plugin:4.6.2:pull (default-cli) @ standalone-pom ---
[INFO] Please report Zanata bugs here: https://zanata.atlassian.net/
[INFO] Loading project config from /home/sdickers/projects/sjd78-ovirt-web-ui/zanata.xml
User config file '/home/sdickers/.config/zanata.ini' not found; ignoring.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.007 s
[INFO] Finished at: 2020-10-14T10:07:15-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.zanata:zanata-maven-plugin:4.6.2:pull (default-cli) on project standalone-pom: Zanata mojo exception: RESTEASY004655: Unable to invoke request: NullPointerException -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Moving zanata.ini back and it works again:
```
 sdickers@ghost  ~/projects/sjd78-ovirt-web-ui  mv ~/.config/zanata.ini{_,}
 sdickers@ghost  ~/projects/sjd78-ovirt-web-ui  yarn intl:pull
yarn run v1.22.5
$ scripts/intl/intl-zanata-pull.sh
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] --- zanata-maven-plugin:4.6.2:pull (default-cli) @ standalone-pom ---
[INFO] Please report Zanata bugs here: https://zanata.atlassian.net/
[INFO] Loading project config from /home/sdickers/projects/sjd78-ovirt-web-ui/zanata.xml
[INFO] Loading user config from /home/sdickers/.config/zanata.ini
[INFO] client API version: 4.6.2, server API version: 4.6.2
[INFO] Zanata command: pull
[INFO] Server: https://zanata.phx.ovirt.org/
[INFO] Project: ovirt-web-ui
[INFO] Version: 1.6
[INFO] Username: sdickers
[INFO] Project type: gettext
[INFO] Enable modules: false
[INFO] Using ETag cache: true
[INFO] Purging ETag cache beforehand: false
[INFO] Locales to pull: [de, es, fr, it, ja, ko, pt-BR, zh-Hans-CN(zh-CN), cs]
[INFO] Encode tab as \t: true
[INFO] Current directory: /home/sdickers/projects/sjd78-ovirt-web-ui
[INFO] Pulling target documents (translations) only
[INFO] Target-language base directory (translations): extra/from-zanata
[INFO] Minimum accepted translation percentage (message based): 0%
[INFO] Create skeletons for untranslated messages/files: false
[INFO] Approved translations only: false
[INFO] Pulling 2 of 2 docs for this module from the server
[?] This will overwrite/delete any existing translations in the above directory.

Are you sure (y/n)?  
```